examples/server-side-rendering/src/polyfills.ts

```diff
@@ -39,7 +39,6 @@
 
 /** Evergreen browsers require these. **/
 import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
```

Comment:
This was removed to the migration.
It's is expected when [upgrading to v7](https://blog.angular.io/version-7-of-angular-cli-prompts-virtual-scroll-drag-and-drop-and-more-c594e22e7b8c#f14f)